src/views/AddTimer.tsx

```diff
@@ -0,0 +1,132 @@
+        const timerType = type as 'stopwatch' | 'countdown' | 'xy' | 'tabata';
+
+        const newTimer: Timer = {
+            id: nanoid(),
+            type: timerType,
+            config: type === "tabata"
+                ? { ...config, initialTime: config.workTime * 1000, isWorkPhase: true }
+                : { ...config, initialTime: type === "stopwatch" ? 0 : totalTime },
+            state: "notRunning",
+            timeLeft: type === "stopwatch" ? 0 : type === "tabata" ? config.workTime * 1000 : totalTime,
+        };
+        addTimer(newTimer);
+        navigate("/");
+    };
+
+
+    return (
+        <FormContainer>
+            <h2>Add New Timer</h2>
+            <form onSubmit={handleSubmit}>
+                <StyledLabel>
+                    Type:
+                    <select value={type} onChange={handleTypeChange}>
+                        <option value="stopwatch">Stopwatch</option>
+                        <option value="countdown">Countdown</option>
+                        <option value="xy">XY</option>
+                        <option value="tabata">Tabata</option>
+                    </select>
+                </StyledLabel>
+                {(type === "countdown" || type === "stopwatch") && (
+                    <>
+                        <StyledLabel>
+                            Hours:
+                            <input name="hours" type="number" placeholder="Hours" value={config.hours} onChange={handleInput} />
+                        </StyledLabel>
+                        <StyledLabel>
+                            Minutes:
+                            <input name="minutes" type="number" placeholder="Minutes" value={config.minutes} onChange={handleInput} />
+                        </StyledLabel>
+                        <StyledLabel>
+                            Seconds:
+                            <input name="seconds" type="number" placeholder="Seconds" value={config.seconds} onChange={handleInput} />
+                        </StyledLabel>
+                    </>
+                )}
+                {type === "xy" && (
+                    <>
+                        <StyledLabel>
+                            Minutes:
+                            <input name="minutes" type="number" placeholder="Minutes" value={config.minutes} onChange={handleInput} />
+                        </StyledLabel>
+                        <StyledLabel>
+                            Seconds:
+                            <input name="seconds" type="number" placeholder="Seconds" value={config.seconds} onChange={handleInput} />
+                        </StyledLabel>
+                        <StyledLabel>
+                            Rounds:
+                            <input name="numberOfRounds" type="number" placeholder="Rounds" value={config.numberOfRounds} onChange={handleInput} />
+                        </StyledLabel>
+                    </>
+                )}
+                {type === "tabata" && (
```

Comment:
Add some guards to make sure users cannot enter bad inputs (or omit required inputs). For Tabata, I was able to not enter any numbers
<img width="391" alt="image" src="https://github.com/user-attachments/assets/087d66a7-24e0-43ba-a967-301ce570b21b">